eclipse-ankaios/README.md

```diff
@@ -71,15 +71,15 @@ Afterwards, open a new terminal inside the devcontainer in VSCode.
 Navigate to the subfolder containing this README file and run the following command to build the devcontainer image:
 
 ```shell
-docker build -t custom-ankaios-dev:0.1 --target dev -f .devcontainer/Dockerfile .
+docker build -t ankaios-blueprint:0.1 -f .devcontainer/Dockerfile .
 ```
 
 Todo! Fix mountpoints, when new folder structure of Microsoft is available..
 
 Start the devcontainer with the required mount points:
 
 ```shell
-docker run -it --privileged --name custom_ankaios_dev -v <absolute/path/to>/blueprint/eclipse-ankaios:/workspaces/app -v <absolute/path/to>/blueprint/in-vehicle-stack:/workspaces/app/in-vehicle-stack -p 25551:25551 --workdir /workspaces/app custom-ankaios-dev:0.1 /bin/bash
+docker run -it --privileged -p 25551:25551 --name ankaios_blueprint --workdir /workspaces/blueprint -v /<absolute-path-to>/software-orchestration/eclipse-ankaios:/workspaces/blueprint ankaios-blueprint:0.1
```

Comment:
I starting to get confused by the actual name of this blueprint which I assumed would be either software-orchestration or maestro. Here you seem to reference with ankaio_blueprint or simply "blueprint". To my understanding this software orchestration blueprint is not exclusive to ankaios and there are other blueprints so it would be great to avoid confusion for people working on multiple blueprints too.

Reply:
We can rename it. The name should be kept general as there are different scenarios later. Then I suggest to mount it under "/workspaces/software-orchestration".

Reply:
@eriksven: It is renamed now and we made some other improvements inside the documentation and Dockerfile. Pls have a look again.

Reply:
Looks good to me, thank you!